config/config.proto

```diff
@@ -53,6 +53,11 @@ message FilterChain {
     // Optional.
     Match match = 2;
 
+    // Global configuration of this filter chain. It will be dismissed when OIDC config set per filter.
```

Comment:
```suggestion
    // Global configuration of this filter chain. It will be ignored when OIDC config set per filter.
```

I assume it's either global or individual today, with no support for overrides yet?

Reply:
I think that it should be overridden by individual config if that exists.